pkg/apis/azuredisk/v1beta2/types.go

```diff
@@ -521,3 +521,36 @@ type ListSnapshotsResult struct {
 	// +optional
 	NextToken string `json:"next_token,omitempty"`
 }
+
+type AzDiskDriverConfiguration struct {
+	Controller ControllerConfiguartion `json:"controller"`
+	Node NodeConfiguration `json:"node"`
+	// The namespace where driver related custom resources are created
+	DriverObjectNamespace string `json:"diverObjectNamespace"`
+	// Boolean field to indicate whether this instance is being used for sanity or integration tests
+	IsTestRun bool `json:"isTestRun"` // not sure if it belongs to controller
+}
+
+type ControllerConfiguartion struct {
+	// Boolean field to indicate this instance is running as controller
+	IsControllerPlugin bool `json:"isControllerPlugin"`
+	// The duration that non-leader candidates will wait to force acquire leadership
+	ControllerLeaseDurationInSec int `json:"controllerLeaseDurationInSec"`
```

Comment:
We can omit `Controller` in the field names since its implied by the configuration object and field names.